src/app/asset-page/artstor-viewer/artstor-viewer.component.html

```diff
@@ -1,4 +1,40 @@
 <div class="asset-viewer" [class.fullscreen]="isFullscreen" [class.multi-view]="isMultiView" (contextmenu)="disableContextMenu($event)">
+  <!-- Large thumbnail fallback
+      - Thumbnail is always showing, and hidden by other viewers placed on top
+      - This allows search crawlers to find the thumbnail
+      -->
+  <div id="fallbackViewer">
+            <img class="item item--thumbnail" alt="{{ asset.title }}" [attr.src]="asset.thumbnail_url"
+                 (error)="asset.fallbackThumbnailUrl()">
+  </div>
+  <div class="more-results overlay-wrap--center" id="loader-wrapper" *ngIf="state == 0"><i
+    class="icon-asterisk icon-spin icon-large"></i></div>
+  <!-- IIIF Image Viewer-->
+  <div [class.aiw-hidden]="state != 1">
+    <div class="item osd-wrap" [attr.id]="osdViewerId" [attr.zoomValue]="lastZoomValue" role="application" attr.aria-label="Interactive image viewer. Now viewing {{ asset.title }}."></div>
+  </div>
```

Comment:
This is a copy/paste and some slight reformatting from the block below. We also added this role and label